### PR TITLE
Kube burner test

### PR DESCRIPTION
**Pull request by dustinblack**

### CODE REVIEW

File: kube-burner-test/config-podman.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+log:
+  level: debug
+logged_outputs:
+  error:
+    level: error
+deployer:
+  type: docker
```
Comment (Harshith-umesh) [inline kube-burner-test/config-podman.yaml]: should this be podman?

Comment (dustinblack) [reply kube-burner-test/config-podman.yaml]: Good catch. I must have missed hitting save.

Comment (dustinblack) [reply kube-burner-test/config-podman.yaml]: Fixed.

**Comment by Harshith-umesh** [comment]
LGTM!

**Pull request merged**